src/components/ui/main-nav.tsx

```diff
@@ -9,11 +9,13 @@ export function MainNav({
   ...props
 }: React.HTMLAttributes<HTMLElement>) {
   const pathname = usePathname()
+  const userRoleURLSegment = pathname.split('/')[1]
+  
   const navLinks = [
-    {name: 'Assessments', href: '/a/assessments'},
-    {name: 'Quizzes', href: '/a/quizzes'},
-    {name: 'Users', href: ''},
-    {name: 'Settings', href: '/a/settings'},
+    {name: 'Assessments', href: `/${userRoleURLSegment}/assessments/`},
```

Comment:
If you login as a Candidate, can you navigate to the Admin route by typing /a/assessments in the browser address bar?

Need to make sure Candidate role cannot access Admin pages.